session_0/item.rb

```diff
@@ -41,5 +53,17 @@ def discounted?
   # Note: If there are no items for category, stock price for category
   # should be zero.
   def self.stock_price_by_category(items)
+    stocks={
```

Comment:
This looks okay right now, but what if there were 100 categories? Could you still write them hand out one-by-one?